lib/licenses.nix

```diff
@@ -545,6 +545,13 @@ lib.mapAttrs mkLicense ({
     free = false;
   };
 
+  firefox = {
```

Comment:
Given that this license is only used in one place, I think it would make more sense to inline it, especially to avoid it being accidentally applied to the source-built Firefox packages.

Reply:
Inlined.